packages/dso-toolkit/components/Voorbeelden/20-Toepassingen/40-Mijn-Omgevingsloket/Project overzicht.config.yml

```diff
@@ -15,3 +15,16 @@ context:
         active: true
   appheading:
     title: Boom kappen in de achtertuin
+  linkList:
+    links:
+      - label: Ingediende verzoeken
+        url: '#'
+        selected: true
```

Comment:
selected -> is current misschien gewoon een duidelijker benaming van deze eigenschap?

Reply:
Mee eens